elasticdl/python/common/args.py

```diff
@@ -561,3 +561,10 @@ def _str_ignore_none(s):
         "--" + k if i % 2 == 0 else k for i, k in enumerate(arguments)
     ]
     return arguments
+
+
+def remove_end_slash(val):
+    """Remove the end slash if it has"""
+    if val and val.endswith("/"):
```

Comment:
Can simplify to `if val.endswith("/"):` since `"".endswith("/")` would return `False` already